content/it/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Comunicazione Piano di controllo - Nodo
+content_type: concept
+weight: 20
+aliases:
+- master-node-communication
+---
+
+<!-- overview -->
+
+Questo documento cataloga le connessioni tra il piano di controllo (_control-plane_), in realtà l'apiserver, e il cluster Kubernetes. L'intento è di consentire agli utenti di personalizzare la loro installazione per rafforzare la configurazione di rete affinché il cluster possa essere eseguito su una rete pubblica (o su IP completamente pubblici resi disponibili da un fornitore di servizi cloud).
+
+
+<!-- body -->
+
+## Dal Nodo al control-plane{#dal-nodo-al-control-plane}
+Kubernetes adotta un pattern per le API di tipo _"hub-and-spoke"_. Tutte le chiamate delle API eseguite sui vari nodi sono effettuate verso l'apiserver (nessuno degli altri componenti principali è progettato per esporre servizi remoti). L'apiserver è configurato per l'ascolto di connessioni remote su una porta HTTPS protetta (443) con una o più forme di [autenticazioni client](/docs/reference/access-authn-authz/authentication/) abilitate. Si dovrebbero abilitare una o più forme di [autorizzazioni](/docs/reference/access-authn-authz/authorization/), in particolare nel caso in cui siano ammesse [richieste anonime](/docs/reference/access-authn-authz/authentication/#anonymous-requests) o [_token_ legati ad un account di servizio (_service account_)](/docs/reference/access-authn-authz/authentication/#service-account-tokens).
+
+Il certificato pubblico (_public root certificate_) relativo al cluster corrente deve essere fornito ai vari nodi di modo che questi possano connettersi in modo sicuro all'apiserver insieme alle credenziali valide per uno specifico _client_. Ad esempio, nella configurazione predefinita di un cluster [GKE](https://cloud.google.com/kubernetes-engine?hl=it), le credenziali del client fornite al kubelet hanno la forma di un certificato client. Si veda
+[inizializzazione TLS del kubelet TLS](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/) per la fornitura automatica dei certificati client al _kubelet_.
+
+I _pod_ che desiderano connettersi all'apiserver possono farlo in modo sicuro sfruttando un account di servizio in modo che Kubernetes inserisca automaticamente il certificato pubblico di radice e un token valido al portatore (_bearer token_) all'interno _pod_ quando questo viene istanziato.
```

Comment:
nit (OK to skip fixing this, but nice to have):
```suggestion
I _Pod_ che desiderano connettersi all'apiserver possono farlo in modo sicuro sfruttando un account di servizio in modo che Kubernetes inserisca automaticamente il certificato pubblico di radice e un token valido al portatore (_bearer token_) all'interno _Pod_ quando questo viene istanziato.
```
because pod is not a common word in Italian.

Reply:
ok to fix it nop. English term will be always Pascal-Case regardless the context or that rule is only related to "k8s Resource"?

Reply:
In _English_ I can choose between (eg) “Service” (the resource name) and “service” and not have to worry too much about which I mean. For a localization you usually have to be a little more precise because the localized term has a different name.

Reply:
according to this rule, the section "### Dall'apiserver ai nodi, Pod, e servizi{#dal-apiserver-ai-nodi-pod-servizi}" should use "Service" (as k8s resource) instead of its the translation "servizi" and so on within the description. @sftim what do you think?

Reply:
I wasn't sure whether the English original was talking about services or Services, so I don't mind either way.

Reply:
Does the API server ever send messages _to_ a Service?